### PR TITLE
[python] Addition of a mass NIfTI pic creation script

### PR DESCRIPTION
**Pull request by cmadjar**
### Description

This adds a `mass_nifti_pic.py` script to allow mass creation of NIfTI files pic rendering stored in the `files` table.

### How to test

Insert a `.nii.gz` file into the `files` table and run `mass_nifti_pic.py` on the `FileID` associated to that file. 
Note: make sure you have all python dependencies install (check the install script for the list of `pip` packages that are required by LORIS-MRI)
```
python mass_nifti_pic.py -p database_config.py -s 10000 -l 10000 -v
```

### CODE REVIEW

**Comment by cmadjar** [comment]
@xlecours Before the holidays, we talked about a script that would create the pic rendering of NIfTI files. The script in that PR will do exactly that. If I remember correctly, you need this for one project so I assigned the review to you. Let me know if you have any question. Thanks!

**Comment by xlecours** [comment]
That is a huge step toward uploading BIDS datasets. Thank you :)

File: python/mass_nifti_pic.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+
+"""Script to mass create the pic images of inserted NIfTI files."""
+
+import os
+import sys
+import getopt
```
Comment (cmadjar) [inline python/mass_nifti_pic.py]: import re

**Comment by jesscall** [comment]
I cannot attest to the code because I am not familiar with python, but manual test is good :+1:

File: python/mass_nifti_pic.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+        return
+
+    # grep the time length from the NIfTI file header
+    is_4d_dataset = False
+    length_parameters = imaging.get_nifti_image_length_parameters(data_dir + nii_file_path)
+    if len(length_parameters) == 4:
+        file_parameters['time'] = length_parameters[3]
+        is_4d_dataset = True
+
+    # grep the SessionID of the file
```
Comment (xlecours) [inline python/mass_nifti_pic.py]: CandID, not SessionID

Comment (cmadjar) [reply python/mass_nifti_pic.py]: @xlecours corrected the comment in the code. Thanks!

**Pull request merged**